spec/services/send_applications_to_provider_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.describe SendApplicationsToProvider do
```

Comment:
Could this spec be a cucumber table? Or perhaps delegate the business-rules bit to cucumber, and test whatever else it does assuming it's got an application it definitely ought to send — maybe there's a case for making this a `SendApplicationToProvider`, having it always send what it's given, and pulling the knowledge of which applications to send up into the caller?